props/blaster.h

```diff
@@ -260,9 +266,39 @@ class Blaster : public PROP_INHERIT_PREFIX PropBase {
     }
   }
 
+  RefPtr<BufferedWavPlayer> tmp;
+  uint32_t len;
+
   void Loop() override {
     PropBase::Loop();
     PollNextAction();
+	  
+    STATE_MACHINE_BEGIN();
```

Comment:
You don't need a state machine here.

Your state machine runs one loop every time Loop() is called, which is what the code would do even if it wasn't in a state machine.